Library/Formula/camlp5.rb

```diff
@@ -13,20 +16,36 @@ class Camlp5 < Formula
 
   deprecated_option "strict" => "with-strict"
   option "with-strict", "Compile in strict mode"
+  option "with-doc", "Install the html and info documentation"
+  option "with-tex", "Install the pdf, ps, and tex documentation"
 
   depends_on "ocaml"
+  depends_on "gnu-sed" => :build if build.with?("doc") || build.with?("tex")
+  depends_on :tex => [:build, :optional]
+  depends_on "ghostscript" => :build if build.with?("tex")
 
   def install
-    if build.with? "strict"
-      strictness = "-strict"
-    else
-      strictness = "-transitional"
-    end
+    args = ["--prefix", prefix, "--mandir", man]
+    args << "--transitional" if build.without? "strict"
 
-    system "./configure", "-prefix", prefix, "-mandir", man, strictness
-    # this build fails if jobs are parallelized
-    ENV.deparallelize
+    system "./configure", *args
     system "make", "world.opt"
     system "make", "install"
+
+    if build.with?("doc") || build.with?("tex")
+      ENV.prepend_path "PATH", Formula["gnu-sed"].opt_libexec/"gnubin"
```

Comment:
Some scripts check for `gsed` as well as `sed` to find `gnu-sed` so it isn't always necessary to prepend the `$PATH` with the directory containing the `sed` executable.

We _could_ do something about this in `superenv` but given how limited in terms of internal use `gnu-sed` is, the already-available documented workaround and requiring a change in default behaviour I'm not entirely convinced on the need.